### PR TITLE
beardofedu's comment change

### LINKED ISSUE #12: beardofedu's checklist
During class I'm going to:

- [x] Create an issue
- [x] Create a branch
- [x] Make changes to our file
- [x] Make commits
- [x] Make a pull request 
- [x] Perform a peer review
- [x] Merge

### PR DESCRIPTION
**Pull request by beardofedu**
Making changes to the `beardofedu` _post file. 

These changes are based on and closes #12

### CODE REVIEW

File: _posts/0000-01-11-beardofedu.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-![electrocat](https://octodex.github.com/electrocat.png)
+![electrocat](https://octodex.github.com/images/electrocat.png)
 {: .center}
 
-Caption here
+I get all charged up when I merge a pull request! :battery:
```
Comment (SirDisney13) [inline _posts/0000-01-11-beardofedu.md]: Nice comment!

File: _posts/0000-01-11-beardofedu.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-![electrocat](https://octodex.github.com/electrocat.png)
+![electrocat](https://octodex.github.com/images/electrocat.png)
```
Comment (SirDisney13) [inline _posts/0000-01-11-beardofedu.md]: Good find noticing that "images" was missing from the URL.

File: _posts/0000-01-11-beardofedu.md (hunk 1 above)
Comment (SirDisney13) [inline _posts/0000-01-11-beardofedu.md]: Funny stuff!

**Review by SirDisney13 — CHANGES_REQUESTED** [review]
I am adding a change request just for the heck of it.  There is nothing to change really, unless you want to pick a different image and come up with another funny comment.

File: _posts/0000-01-11-beardofedu.md (hunk 2 above)
Comment (beardofedu) [reply _posts/0000-01-11-beardofedu.md]: 😍 
 👍

File: _posts/0000-01-11-beardofedu.md (hunk 1 above)
Comment (beardofedu) [reply _posts/0000-01-11-beardofedu.md]: 🤣

**Comment by beardofedu** [comment]
@SirDisney13 made the requested changes, need the 👍

**Review by SirDisney13 — APPROVED** [review]

**Pull request merged**